Functions/Quantification/xASL_quant_CBF.m

```diff
@@ -41,39 +43,59 @@
 % -----------------------------------------------------------------------------------------------------------------------------------------------------
 % EXAMPLE: [ScaleImage, CBF] = xASL_quant_CBF(PWI, M0_im, imSliceNumber, x);
 % __________________________________
-% Copyright 2015-2019 ExploreASL
+% Copyright (c) 2015-2023 ExploreASL
 
 
 %% Admin
-fprintf('%s\n','Quantification CBF single PLD:');
+if x.modules.asl.bMultiPLD
+	fprintf('%s\n', 'Performing multi-PLD quantification');
+else
+	fprintf('%s\n', 'Performing single-PLD quantification');
+end
+
+if nargin < 4
+	error('Four input parameters required.');
+end
 
 if  xASL_stat_SumNan(M0_im(:))==0 && x.modules.asl.ApplyQuantification(5)
     error('Empty M0 image, something went wrong in M0 processing');
+elseif xASL_stat_SumNan(M0_im(:))==0
+    warning('Weird M0 image detected');    
 end
 
 if nargin<5 || isempty(bUseBasilQuantification)
```

Comment:
Currently, we never check if a boolean really has a 0 or a 1. Shall we create an issue for Maarten, to check for all input parameters of ExploreASL, to verify their content (if they exist)? I wouldn't make this too complicated, we could later of course add dependencies, but for now it would already be nice to check that input parameters are correct before users get weird pipeline results down the line.

Reply:
You mean validating of studyPar.json and dataPar.json? No we don't have that, we can create a new issue...